vars/s2i-ruby-26-container-rhel8.yml

```diff
@@ -0,0 +1,6 @@
+deployment: "oc new-app registry.access.redhat.com/ubi8/ruby-26~https://github.com/sclorg/s2i-ruby-container.git --context-dir=2.5/test/puma-test-app/"
```

Comment:
image version is 2.6, while context-dir is 2.5

Reply:
Fixes